File: launchdarkly/typescript-axios/src/GetRootExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.OtherApi(configuration).getRoot().then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling OtherApi#getRoot:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});

File: launchdarkly/ruby/src/GetRootExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["ApiKey"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::OtherApi.new.get_root

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling OtherApi#get_root: #{e}"
end
